src/elements.js

```diff
@@ -13,6 +13,12 @@ const directUpdater = {
  * @typedef { import('../../types/options').AnnotationPluginOptions } AnnotationPluginOptions
  */
 
+/**
+ * @param {string} prop
+ * @returns {boolean}
+ */
+export const isIndexable = (prop) => prop === 'color' || prop === 'font';
```

Comment:
This should be in the plugin descriptors, unless it does not work there?

For example:
https://github.com/chartjs/Chart.js/blob/cfe8e0311bca46e43d5630e91bf82f470df3f56d/src/plugins/plugin.tooltip.js#L1322-L1335

Reply:
@kurkle it is in element because is also used in element code: https://github.com/chartjs/chartjs-plugin-annotation/pull/801/files#diff-4e8835b7cdbc98ff051ec4984253e2a838ff262a5ec9e3ecf469aa0569cd8429R132

Putting it in annotation and exported to element, we have a cycle in the "import"

Reply:
@kurkle what do you think? Is it acceptable or do you think I must move in annotation.js ?